SmartDisenchantEverything/Program.cs

```diff
@@ -38,10 +39,28 @@ public static void RunPatch(IPatcherState<ISkyrimMod, ISkyrimModGetter> state)
                 )
                     continue;
 
+                // skip armor that is non-playable
+                if (item is IArmorGetter armorGetter && armorGetter.MajorFlags.HasFlag(Armor.MajorFlag.NonPlayable)) continue;
+
+                // skip weapon that is non-playable
+                if (item is IWeaponGetter weaponGetter && weaponGetter.MajorFlags.HasFlag(Weapon.MajorFlag.NonPlayable)) continue;
+
                 // check if item has keywords
                 if (item is not IKeywordedGetter itemKeywords || itemKeywords.Keywords == null)
                     continue;
 
+                // skip if item has blacklisted keywords
+                bool skip = false;
```

Comment:
Can do this more elegantly by just doing `if (itemKeywords.Keywords.Any(e => Settings.KywdBlacklist.Contains(e))) continue;`

Reply:
Thanks for the tip! Not a CS guy myself, so good to know you can do stuff like that.

Added that to my branch.

Reply:
Tbh I'm not either, it's just tricks I picked up along the way haha this is all just to get some practice :)

Was there any reason to skip non playable equipment other than to speed up the patcher?

Reply:
lol fair enough

And yep, pretty much just that - if you (the player) can't see it, why touch it. That and I've got so many other patchers going if I can skip some masters I'm less likely to hit the plugin master limit (since I've already had to tweak some of the other patchers I use to skip records I don't need patched just to get it back below).